shared/keycodes.ts

```diff
@@ -0,0 +1,11 @@
+interface KeyMapping {
+  [key: string]: number;
+}
+
+const KeyCodes: KeyMapping = {
+  ENTER: 13,
+  UP_ARROW: 38,
+  DOWN_ARROW: 40
+};
+
+export default KeyCodes;
```

Comment:
React provides some more readable keyboard event attributes that don't require the mapping https://reactjs.org/docs/events.html#keyboard-events

Reply:
:+1: